### DISCUSSION TITLE
Which systems in the Dirt Belt should the Free Worlds patroll as part of the Syndicate deal?

### DISCUSSION THREAD

**Discussion by hexagonrecursion** [endless-sky/endless-sky#9375] [General]
- The dialogue in the "FW Syndicate Diplomacy 1B" mission promises that the Free Worlds will defend Syndicate ships going to/from Tarazed. (introduced in d20b245d9ef40b02ea52fcc39bea84b19c6847e0 by @mzahniser)
- The dialogue in the "FW Early Warning 2" mission mentions that the Free Worlds patrols and the Navy patrols in the eastern Dirt Belt do not shot at each other (an unofficial ceasefire) (introduced in 9eafe444565ad573370ac445bcd0949243ac76fd by @Amazinite)

But both are currently just flavor text. No fleets are deployed or changed. I have found two event placeholders:

https://github.com/endless-sky/endless-sky/blob/a4bdd1be927d2ac71783c14ee5b667c323a76738/data/human/campaign%20events.txt#L997-L1000

https://github.com/endless-sky/endless-sky/blob/a4bdd1be927d2ac71783c14ee5b667c323a76738/data/human/campaign%20events.txt#L1027-L1030

I would like to work on filling this in, but I am having trouble figuring out precisely which systems the Free Worlds should patrol.
1. Just the trade route from the Syndicate to Tarazed?
2. The entire Dirt Belt?
3. The eastern Dirt Belt? Where to draw the line between it and the rest of the Dirt Belt?

The trade route: Menkent, Cebalrai, Rasalhague, Ascella, Peacock, Kaus Australis, Dabih. I did not include Vega and Altair because they are not part of the dirt belt. I would also add the Tarazed system itself. It is not part of the Dirt Belt, but it is not defended by the Navy

![image](https://github.com/endless-sky/endless-sky/assets/52621858/1b6be6c9-a626-4d2f-9213-56cab3a8155b)

How do the Free Worlds' patrol fleets get to/from this systems? There is at least one more system governed by the empire that they would have to pass trough. Albaldah?

The dialogue in the "FW Early Warning 2" mission mentions "Navy and militia patrols" on Harmony. This sounds like Girtab should also be patrolled.

@mzahniser, @Amazinite, does this align with your vision? Did I miss any systems that should also be patrolled by the Free Worlds?

**Comment by bene-dictator** [comment]
Free Worlds ships would not go within two jumps of Sol. Northern politicians wouldn't let that happen.

The systems from Rasalhague to Dabih have militias defending them, presumably they'd just allow Free Worlds patrols / the militia fleets of Wayfarer and New Britain to land at their bases. This would set up the later expansion into that area by the Free Worlds. The planets in that system are (as far as I'm concerned) affiliated with Tarazed Corp. because they're a major customer of the minerals mined on those planets.

Girtab has a militia but was not part of the Southern Mutual Defence Pact so it didn't join the Free Worlds. After the Free Worlds forms, Free Worlds ships have to pass this system to get to Tarazed in order to get Tarazed Corporation's support so it should already get patrols.

(This is all my opinion, by the way.)

**Reply by hexagonrecursion** [reply]
I see. Menkent and Cebalrai are indeed too close to Sol.
